src/librustc/ty/adjustment.rs

```diff
@@ -36,13 +36,13 @@ use ty::subst::Substs;
 /// sized struct to a dynamically sized one. E.g., &[i32; 4] -> &[i32] is
 /// represented by:
 ///
-/// ```
-/// Deref(None) -> [i32; 4],
-/// Borrow(AutoBorrow::Ref) -> &[i32; 4],
-/// Unsize -> &[i32],
-/// ```
+///    ```
+///    Deref(None) -> [i32; 4],
+///    Borrow(AutoBorrow::Ref) -> &[i32; 4],
+///    Unsize -> &[i32],
+///    ```
 ///
-/// Note that for a struct, the 'deep' unsizing of the struct is not recorded.
+///    Note that for a struct, the 'deep' unsizing of the struct is not recorded.
```

Comment:
Why these need to be indented?

Reply:
With pulldown, these paragraphs need to be indented to be considered continuations of the list, and to avoid splitting the list into two parts.

Reply:
I see, the diff doesn't indicate there's a list. Though I'd expect the whole block to be indented for each list item:

```
1. Something
   like
   this
```

(Not sure about the official style though.)